app/controllers/api/v1/posts_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Api
+  module V1
+    class PostsController < ApplicationController
+      def index
+        user = User.find(params[:user_id])
+        posts = user.posts
+        render json: posts
+      end
+    end
+  end
+end
```

Comment:
- [ ] I was unable to list the posts on the postman. For the project you can skip the authentication if you have not set it up.

  ```
  skip_before_action :authenticate_user!
  ```

  ![image](https://github.com/mershark/blog-app/assets/54848363/fe2a253d-7883-4527-a53a-bff9127beee2)